desk

```diff
@@ -114,56 +114,95 @@ cmd_run() {
 }
 
 
+# Usage:        desk list [options]
+# Description:  List all desks along with a description
+# --only-names: List only the names of the desks
+# --no-format:  Use ' - ' to separate names from descriptions
 cmd_list() {
     if [ ! -d "${DESKS}/" ]; then
-        echo "No desk dir! Run 'desk init'." 
+        echo "No desk dir! Run 'desk init'."
         exit 1
     fi
+    local SHOW_NAMES DESKEXT AUTO_ALIGN name desc len longest out
+
+    while [[ $# -gt 0 ]]; do
+        case "$1" in
+            --only-names) test "$SHOW_NAMES" != false && SHOW_DESCRIPTIONS=false && AUTO_ALIGN=false ;;
```

Comment:
when would this `test` ever not return `0`?

can't this just be 
```sh
--only-names) SHOW_DESCRIPTIONS=false ;;
```
for the same effect? `SHOW_NAMES` can then be removed entirely.